src/main/java/com/crowdin/cli/client/CrowdinClientTask.java

```diff
@@ -0,0 +1,39 @@
+package com.crowdin.cli.client;
+
+import com.crowdin.client.tasks.model.AddTaskRequest;
+import com.crowdin.client.tasks.model.Status;
+import com.crowdin.client.tasks.model.Task;
+
+import java.util.List;
+
+public class CrowdinClientTask extends CrowdinClientCore implements ClientTask {
+
+    private final com.crowdin.client.Client client;
+    private final String projectId;
+
+    public CrowdinClientTask(com.crowdin.client.Client client, String projectId) {
+        this.client = client;
+        this.projectId = projectId;
+    }
+
+    @Override
+    public List<Task> listTask() {
+        return executeRequestFullList((limit, offset) -> this.client.getTasksApi()
+            .listTasks(Long.valueOf(projectId), limit, offset, null));
+    }
+
+    @Override
+    public List<Task> listTask(Status status) {
+        return executeRequestFullList((limit, offset) -> this.client.getTasksApi()
+                .listTasks(Long.valueOf(projectId), limit, offset, status));
```

Comment:
Wrong indentations here and in the method below

Reply:
done